cleanup-closed-pull-requests/src/branches.ts

```diff
@@ -1,6 +1,7 @@
 import * as core from '@actions/core'
 import * as github from '@actions/github'
 import { GitHub } from '@actions/github/lib/utils'
+import { extractPullRequestNumber } from './applications'
```

Comment:
Use this: https://github.com/quipper/monorepo-deploy-actions/blob/1bcb23124007404147bd683beee770b0456fe20d/cleanup-closed-pull-requests/src/applications.ts#L129-L139